stix_shifter_modules/guardium/stix_translation/query_constructor.py

```diff
@@ -551,7 +635,7 @@ def _parse_expression(self, expression, qualifier=None) -> str:
                 return "{}".format(comparison_string)
 
         elif isinstance(expression, CombinedComparisonExpression):
-            operator = self.comparator_lookup[str(expression.operator)]
```

Comment:
Keep as is. This line shouldn't have been changed.

Reply:
Done the change.